Dockerfile

```diff
@@ -15,7 +16,9 @@
 
 # Pull the base image.
 FROM alpine:latest
-COPY requirements.txt .
+WORKDIR /home/spiderfoot
+COPY . .
+ENV SPIDERFOOT_VERSION 3.0.0
```

Comment:
I guess if we're going with using the Dockerfile to always produce from what's local instead of the master branch, we should remove this version seeing as we won't know what the verison will be.

Reply:
@ikp4success why are you removing the `COPY requirements.txt .`?
https://stackoverflow.com/a/34399661/1831675